alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
```

Comment:
Yep - you got it.  Just to clarify - we use `rem` in cases where we'd want the dimension to scale if the user chooses a different font-size (from Account Settings).  Those base font-sizes are : 18px (small), 20px (medium/normal), 22px (large), and 24px (huge).  Of course, we calculate all `rem` values based on the 20px.